packages/cloudflare-workers-bindings-extension/package.json

```
@@ -69,7 +69,8 @@
 		"eslint": "^8.57.0",
 		"npm-run-all": "^4.1.5",
 		"typescript": "^5.4.5",
-		"vsce": "^2.15.0"
+		"vsce": "^2.15.0",
+		"wrangler": "^3.95.0"
```

Can we use the workspace version instead? Just so I can use the new APIs from wrangler before it is released 🤔 

```suggestion
		"wrangler": "workspace:*"
```